Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.1.2'
+ruby '2.3.1'
```

Comment:
Hi, @mridulnagpal - although I'm not against upgrading this requirement, I don't think we should include it in this topically unrelated pull request. Would you mind opening a new issue/PR for this change, or at least removing it from your PR? I appreciate the thought, but one thing at a time, please!